tests/fsharp/readme.md

```diff
@@ -0,0 +1,15 @@
+# F# Compiler Cross-Platform Test Suite
+
+## Layout
+
+TODO
```

Comment:
Marking as needs fixing until the TODO is gone

Reply:
Ok, I'll put something together.

To describe the layout, we may want to first split the test infrastructure from the folders with individual test files to get better distinction, that's what I faced when I've put TODO + not familiar enough with the code layout 🙂.

Reply:
@smoothdeveloper any progress on this?

Reply:
Came back over the pond to the old continent. When I have some time again, I'll work on fixing remaining comments on https://github.com/dotnet/fsharp/pull/6596#discussion_r283280722.

I don't think it will be during this week.

Reply:
@KevinRansom could you submit an edit to my branch to address the TODO:

> that's what I faced when I've put TODO + not familiar enough with the code layout 

Maybe you are more familiar with this test suite? would you replace it with a brief description of last known state from your side?